app/templates/components/side-bar.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="ui left inverted vertical menu sidebar {{if this.sidebarVisible 'visible'}}">
-    <div class="search-bar d-flex items-center space-between m-2 p-2">
+    <div class="hidden-item search-bar d-flex items-center space-between m-2 p-2">
```

Comment:
Why

Reply:
by mistake, my GitHub branch didn't show the status before.

Reply:
Nope sir. I created a different branch for that. I switched to new branch I remembered. Thanks.